build.sbt

```diff
@@ -1199,7 +1198,7 @@ lazy val sbtClientProj = (project in file("client"))
       s"-H:Name=${target.value / "bin" / "sbtn"}",
     ) ++ (if (isLinux && isArmArchitecture)
             Seq("-H:PageSize=65536") // Make sure binary runs on kernels with page size set to 4k, 16 and 64k
-          else Nil),
+          else Nil) ++ (if (isLinux) Seq("--static", "--libc=musl") else Nil),
```

Comment:
This should be `isLinux && !isArmArchitecture`

Reply:
Oh saw you made the comment in https://github.com/sbt/sbt/pull/7823#discussion_r1814347773

Guess we coincidentally were doing research the same time... again.